modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -0,0 +1,57 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= (Optional) Configuring host network interfaces in the `install-config.yaml` file
+
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState on the `baremetal` network. To use the `networkConfig` configuration setting, you must pass an NMState key file as a YAML string, denoted by the `|` symbol. See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
```

Comment:
@johnwilkins
As of 4.10.0-0.nightly-2022-01-21-231627 networkConfig no longer accepts string format (denoted by the pipe character)
it instead only accepts yaml format.
aside from removing the pipe character, behaviour is the same.

Reply:
fixed